cockatrice/src/carddatabasemodel.h

```diff
@@ -12,7 +12,7 @@ class FilterTree;
 class CardDatabaseModel : public QAbstractListModel {
     Q_OBJECT
 public:
-    enum Columns { NameColumn, SetListColumn, ManaCostColumn, PTColumn, CardTypeColumn, CMCColumn };
+    enum Columns { NameColumn, SetListColumn, ManaCostColumn, PTColumn, CardTypeColumn, ColorColumn };
```

Comment:
@ZeldaZach I'm pretty sure CMC has been asked for if it's not already supported. Want to throw that in too?